packages/generator-langium/templates/packages/language/langium-config.json

```diff
@@ -11,5 +11,5 @@
             "out": "src/syntaxes/<%= language-id %>.monarch.ts"
```

Comment:
Suggestion: As mentioned above, we should try to make this conditional.

Reply:
Yes, true, I will adapt this.

Reply:
Done, see above.